packages/ui/tailwind-design-system.ts

```diff
@@ -124,10 +146,56 @@ export default {
           11: 'hsl(var(--canary-background-11))',
           12: 'hsl(var(--canary-background-12))',
           13: 'hsl(var(--canary-background-13))',
-          surface: 'hsl(var(--canary-background-surface))',
+          // surface: 'hsl(var(--canary-background-surface))',
           danger: 'hsla(var(--canary-background-danger))',
           success: 'hsla(var(--canary-background-success))'
         },
+        'cn-background': {
+          DEFAULT: 'lch(from var(--cn-bg-1) l c h / <alpha-value>)',
```

Comment:
This is how we should add the opacity variant for `lch` format colors, as tailwind CSS doesn't support it out of the box. The long term plan is to remove all opacity variants and replace them with their equivalent solid colors from the design system, once avaiable.